src/test/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryServiceIT.java

```diff
@@ -231,4 +240,76 @@ public void stressTestCreateUrgentEvent()
     }
     sw.stop();
   }
+
+  private void generateDummyException(int value, SFSession session) throws SnowflakeSQLLoggedException
+  {
+    if (value > 0)
```

Comment:
why do you need to check value > 0? If it has to be positive, use `assert`.

Reply:
Took this out